libc/include/CMakeLists.txt

```diff
@@ -19,11 +19,10 @@ add_header(
 
 # TODO: Can we simplify this macro expansion?
 # https://github.com/llvm/llvm-project/issues/117254
-macro(add_header_macro TARGET_NAME YAML_FILE DEF_FILE GEN_HDR DEPENDS)
+macro(add_header_macro TARGET_NAME YAML_FILE GEN_HDR DEPENDS)
```

Comment:
Does this let us simplify the `add_gen_header` fn, as per the TODO above?  Perhaps we can strip out the `DEF_FILE` argument there, too, at least?

Reply:
I don't know much about cmake or the organization of libc's cmake bits.  My intent in this change is only to introduce the cleaned-up CLI for hdrgen and touch cmake only as needed to keep it working as is.  I anticipate many more incremental changes to further clean things up as we refine the feature set and CLI of hdrgen.

Reply:
ah, nevermind! my suggestion was done already in this very PR, sorry I missed it!